modules/store/spec/types/store_module.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { expecter } from 'ts-snippet';
+import { compilerOptions } from './utils';
+
+describe('StoreModule', () => {
```

Comment:
🥇

Reply:
Thanks for adding the label, I always forgot to do that 🤦‍♂️